db/seeds/sample_data.rb

```diff
@@ -99,7 +99,7 @@
   puts
   print "...for user number #{count + 1}/#{user_count} (#{login})"
 
-  user = User.find_by_login(login)
```

Comment:
@NobodysNightmare do you really this change? – `User.find_by_login` is defined in `app/models/user.rb` and makes logins case-insensitive.